data/binding/preference_test.go

```diff
@@ -13,12 +13,13 @@ func TestBindPreference_DataRace(t *testing.T) {
 	a := test.NewApp()
 	p := a.Preferences()
 	key := "test-key"
-	const n = 100
 
+	const n = 100
 	var wg sync.WaitGroup
+	wg.Add(n)
```

Comment:
Isn't this considered bad practice? I thought I read you should only increment as each item is added

Reply:
Anyhow I'd be tempted to say that in a refactor like this I'd prefer to see no test changes ;)

Reply:
Yeah, this seems like some sort of thing that I accidentally committed after some race fix testing. I have no idea what is the best practice. The official example adds one at a time but I also can't find anything to say that adding them all at once would be bad. It is certainly faster 😅